core/src/avm2/object/xml_object.rs

```diff
@@ -115,6 +115,36 @@ impl<'gc> XmlObject<'gc> {
         )
     }
 
+    pub fn elements(
+        &self,
+        name: &Multiname<'gc>,
+        activation: &mut Activation<'_, 'gc>,
+    ) -> XmlListObject<'gc> {
+        let children = if let E4XNodeKind::Element { children, .. } = &*self.node().kind() {
+            children
+                .iter()
+                .filter(|node| node.is_element() && node.matches_name(name))
+                .map(|node| E4XOrXml::E4X(*node))
+                .collect()
+        } else {
+            Vec::new()
+        };
+
+        let list = XmlListObject::new_with_children(
+            activation,
+            children,
+            Some(XmlOrXmlListObject::Xml(*self)),
+            None,
```

Comment:
Shouldn't this use `Some(name)`?

> 13.4.4.13 XML.prototype.elements
> 3. Let list be a new XMLList with list.[[TargetObject]] = x and **list.[[TargetProperty]] = name**

Reply:
I think this was because avmplus never set the target property here:
https://github.com/adobe/avmplus/blob/858d034a3bd3a54d9b70909386435cf4aec81d21/core/XMLObject.cpp#L1695

Reply:
Oh, fun. Add a note?

Reply:
Added a note.